src/components/DateChooser/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { Component } from 'react';
+import { func } from 'prop-types';
+import { DatePicker } from 'material-ui-pickers';
+import DateFnsUtils from 'material-ui-pickers/utils/date-fns-utils';
+import MuiPickersUtilsProvider from 'material-ui-pickers/utils/MuiPickersUtilsProvider';
+import ChevronRightIcon from 'mdi-react/ChevronRightIcon';
+import CalendarIcon from 'mdi-react/CalendarIcon';
+import ChevronLeftIcon from 'mdi-react/ChevronLeftIcon';
+import { date } from '../../utils/prop-types';
+
+/**
+ * Display a date picker modal to select a date.
+ */
+export default class DateChooser extends Component {
```

Comment:
This class seems familiar. Didn't I cover this in another review where this was named `DatePicker`?